build/build-bundle.js

```diff
@@ -45,10 +44,6 @@ const isDevtools = file =>
 /** @param {string} file */
 const isLightrider = file => path.basename(file).includes('lightrider');
 
-const BANNER = `// lighthouse, browserified. ${VERSION} (${COMMIT_HASH})\n` +
-  '// @ts-nocheck\n'; // To prevent tsc stepping into any required bundles.
```

Comment:
just confirming you want this gone?

i saw it in the banner for a bit, but now its totally gone.

Reply:
it only ever worked when the entire bundle was on a single line. instead i just changed the way this bundled file is required in the smoke entry

Reply:
> it only ever worked when the entire bundle was on a single line

`@ts-nocheck` definitely works for multi-line files, we use it in several places for that...

Reply:
ok so it wasn't working because of the `!`, guess it isn't seen as a block comment in ts compiler

![image](https://user-images.githubusercontent.com/4071474/93405388-7e686900-f852-11ea-89f9-e138420ce9bb.png)

Reply:
`browserify-banner` doesn't support not doing `/*!` if you use the banner template file. Are we fine with the `require('' + '....')` hack?